java/data/recipe.mcdoc

```diff
@@ -99,8 +107,11 @@ struct ItemResult {
 }
 
 type Ingredient = (
-	IngredientValue |
-	[IngredientValue] |
+	#[until="1.21.2"] IngredientValue |
+	#[until="1.21.2"] [IngredientValue] |
+	// TODO: does not accept 'air'
```

Comment:
I think this might have been the case before too (but hasn't always), but its now certainly enforced during validation. I don't think mcdoc can handle that currently.

Reply:
Not really a big deal imo

Reply:
Yeah, not a big deal, just wanted point out the added TODO note.